icechunk/src/conflicts/detector.rs

```diff
@@ -16,6 +16,7 @@ use crate::{
 
 use super::{Conflict, ConflictResolution, ConflictSolver};
 
+#[derive(Clone, Debug)]
```

Comment:
Python is annoying to implement without this (accepting a union as argument to method)

Reply:
ups, I forgot to add those